docs/distribution_methods.md

```diff
@@ -1 +1,39 @@
 # Mobile App Distribution Methods
+
+Mobile apps are distributed either through the public app stores or   MDM InTune. Please review the diagram below to determine the distribution method for your app.
+
+![Diagram of the various distribution methods. If the app is for the general public, it is distributed via the Apple App Store and Google Play Stores. If it is an app for Government Employees, it is distributed via MDM inTunes. If it is an app for Employees and Contractors it is distributed as an unlisted app via the Apple App Store](assets/distribution.drawio.svg)
+
+Public apps are released through the Province's accounts on Apple's App Store and Google's Play Store. You must list your app through the Province's accounts. The developer experience team will assist you in setting up your app on these accounts.  Please refer to the [project initiation](initiate.md) page for the information needed to setup an app. Once setup you will have access to [App Store Connect](https://appstoreconnect.apple.com) and [Google Play Console](https://play.google.com/console/about/) to manage your app.
+
+Both the Apple App Store and Google Play Store have an app review process. When creating your app, it is important to consider how the app reviewer will interact with your app. For instance, if your app requires a login, the app reviewer will need either an account or access to a demo account. If your internal app cannot go through the app review process, please inform the [developer experience team](mailto:Developer.Experience@gov.bc.ca).
+
+Please consult [Apple's app guidelines](https://developer.apple.com/app-store/review/) and [Google's app guidelines](https://support.google.com/googleplay/android-developer/answer/9859455?hl=en&ref_topic=7072031&sjid=10634496881788336983-NA) when preparing your app's release. It is important to follow their guidelines to ensure a smooth app review process.
+
+Employee apps are distributed through [MDM inTune by the OCIO Device Management Team](https://citz.sp.gov.bc.ca/sites/ES/DS/MDAS/Docs/SitePages/Home.aspx). While historically employee apps only supported iOS devices, it is possible for Android support. If you are planning to create an internal Android app, please contact us. You will be the first app and will need extra setup.
+
+Employee and contractor apps also need extra setup in Apple's App Store Connect. The developer experience team will assist in this setup.
```

Comment:
we might want a link here as well for those folks that dip in and out of parts of docs and don't read them from top to bottom.

Reply:
Added a link to the contact page